views/attackcost.tmpl

```diff
@@ -0,0 +1,378 @@
+{{define "attackcost"}}
+<!DOCTYPE html>
+<html lang="en">
+{{ template "html-head" "Majority Attack Cost Calculator"}}
+{{template "navbar" . }}
+
```

Comment:
Since there's a lot going on in this file, consider throwing up some section comments

`{{- /* ADJUSTABLE PARAMETERS */ -}}`

The dashes at the beginning and end mean to strip whitespace before and after during template parsing, so not only is the file easier to read, it ends up being smaller.